### ISSUE TITLE
Pass environment variables defined in the pipeline

### ISSUE THREAD

**Issue by kim** [oscoin/ci#5]
The command hook needs to pass variables explicitly to `docker run`. There's no way to distinguish which variables (of `env`) were set in the pipeline, so perhaps require some prefix?

**Comment by kim** [comment]
I came up with this:

```shell
#!/usr/bin/env bash
args=()
for var in ${!BUILD_*}
do
    args+=( "-e${var#BUILD_}=${!var}" )
done
printf "%s\n" "${args[@]}"
```

That is, passing variables from the pipeline requires prefixing their names with `BUILD_`, but that prefix is removed before passing it to docker.

@xla you think that'd work for you?

**Comment by xla** [comment]
Definitely a viable solution.

**Comment by kim** [comment]
Closed via 9fa364cec4b91e663afdf1422521ca5d0fe77c76
